PTN/parse.py

```diff
@@ -98,6 +98,7 @@ def parse(self, name):
             raw = raw[self.start:self.end].split('(')[0]
 
         clean = re.sub('^ -', '', raw)
+        clean = re.sub('www.*.* ', '', clean)
```

Comment:
Doing twice .* Is redundant as it means to match any char at any count.
So basically you strip www*
What about movies/series that start with www?